en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/overriding-endpoints-for-imported-apis.md

```diff
@@ -0,0 +1,47 @@
+# Overriding Endpoints for deployed APIs
+
+The API deployed from the API Manager once deployed in Choreo Connect, it needs to be moved across different environments. When moving from one environment to the other backend service details can be altered. In those cases, we need to override the endpoint details of the API
+The following section describes how this can be overriden on the APIs that are imported from the WSO2 API Manager.
+
+## Overriding endpoints using environment variables 
+
+You can define endpoint details as the Adapter environment variables at docker-composer.yaml file.
+
+1. Set the production or sandbox endpoint(s) of an API.
+
+    ``` java tab="Format"   
+    <API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint-URL>" 
+    ```
+    
+    - `<API-ID>` - ID generated for the imported API. Sample name would be `api_30e623704c5c5479b7c0d9ab78e965df02c1610401e37cbd557e6353e3191c76`. Here the 
+    `<API-ID>` is `api_30e623704c5c5479b7c0d9ab78e965df02c1610401e37cbd557e6353e3191c76`
```

Comment:
This does not explain what 30e623704c5c5479b7c0d9ab78e965df02c1610401e37cbd557e6353e3191c76 means

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/3568/commits/73754297fabf28e4531517f7935baa2c14c1f8ae